sensor/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/board_detectors/dotboard_detector.py

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+# cSpell:enableCompoundWords
```

Comment:
What is this for?

Reply:
It was for xarray before I changed it for x_array

Reply:
I see, thank you. So now will you remove it?

Reply:
Sorry I mistook the line in question. This is due to the dotboard -> dot board

Reply:
I see, thank you.
By the way, just FYI, you can manage your own repository-specific dictionary like as follows.
https://github.com/tier4/CARET_analyze/blob/81dd5ea1c5ebe2e13df5114510d6db42ec9a4789/.cspell.json
https://github.com/tier4/cspell-dicts/blob/91536bc7f02df0807a4cb14fe1083d41c28f344d/.cspell.json

Please consider it if necessary.

Reply:
@aohsato @yabuta 
Should we better use our own dictionary ? Seems better that way no?
We may end up using a lot of calibration-only vocab

Reply:
@knzo25 I agree with maintaining our own dictionary for calibration. Do you want to close this pull request as well? https://github.com/tier4/autoware-spell-check-dict/pull/456

@kenji-miyake Let me make sure. In your example, does the dictionary for CARET additionally apply with https://github.com/tier4/autoware-spell-check-dict same time, correct?

Reply:
@aohsato Exactly. Please refer to the following files.
https://github.com/tier4/CalibrationTools/blob/3ac4928e705e652ff114ea48a4dba046ff622ce0/.github/workflows/spell-check-differential.yaml#L16
https://github.com/autowarefoundation/autoware-github-actions/blob/6d0a13d04a2a73195e5c76c6d7cd91af2450ebd1/spell-check/action.yaml

Reply:
Migrated to using our own dict and now the test are passing in the actions